src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Color[] colorsList = Color.values();
+    private static final Random random = new Random();
```

Comment:
why static? we write our code in OOP style, so its better not to use it, except for constants
```suggestion
    private final Color[] colorsList = Color.values();
    private final Random random = new Random();
```

Reply:
@kerrrusha what is the relationship between creating OOP style code and using static or non-static fields?
I use a static modifier because the values of these variables will not change and the result will always be the same.